### ISSUE TITLE
Makefile Incorrectly maps deploy: @forge create

### ISSUE THREAD

**Issue by caducus7** [Cyfrin/foundry-erc20-cu#15]
I noticed an issue in the Makefile regarding the deploy target. The --broadcast argument is being used incorrectly with forge create. It should be passed through forge script instead. I recommend updating it as follows:
```
deploy:
    @forge script script/DeployCaducusToken.s.sol --rpc-url <your_rpc_url> --private-key <your_private_key> --broadcast
```

Same goes for the rest of the deploy targets.

**Comment by EngrPips** [comment]
Hello @caducus7, Thanks for this contribution. Please make a `PR` to correct the issue.

**Comment by PatrickAlphaC** [comment]
fixed in #16

**Comment by EngrPips** [comment]
Interesting.
